datahub-web-react/src/app/search/SearchHeader.tsx

```diff
@@ -39,43 +53,24 @@ export const SearchHeader = ({
     authenticatedUserPictureLink,
 }: Props) => {
     return (
-        <Header
-            style={{
-                position: 'fixed',
-                zIndex: 1,
-                width: '100%',
-                backgroundColor: 'rgb(51 62 76)',
-                fontSize: '18px',
-                height: '64px',
-                lineHeight: '20px',
-                color: '#fff',
-                padding: '0px 80px',
-            }}
-        >
-            <div style={{ display: 'flex', alignItems: 'center' }}>
-                <Link
-                    style={{
-                        height: '64px',
-                        padding: '15px 30px',
-                        display: 'flex',
-                        alignItems: 'center',
-                    }}
-                    to="/"
-                >
-                    <Image style={{ width: '34px', height: '30px' }} src={GlobalCfg.LOGO_IMAGE} preview={false} />
-                    <div style={{ color: 'white', fontWeight: 'bold', padding: '15px' }}>DataHub</div>
-                </Link>
-                <SearchBar
-                    types={types}
-                    initialQuery={initialQuery}
-                    selectedType={selectedType}
-                    placeholderText={placeholderText}
-                    suggestions={suggestions}
-                    onSearch={onSearch}
-                    onQueryChange={onQueryChange}
-                />
-                <ManageAccount urn={authenticatedUserUrn} pictureLink={authenticatedUserPictureLink} />
-            </div>
+        <Header style={styles.header as any}>
+            <Link to="/">
+                <Space size={4}>
+                    {' '}
```

Comment:
if this space is just for the sake of padding, i'd commend just using css-padding

Reply:
good catch this should not be there.